Android/quickstart-kotlin/app/src/main/java/com/example/immersivereadersdk/WebAppInterface.kt

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.example.immersivereadersdk
+
+import android.content.Context
+import android.webkit.JavascriptInterface
+import android.webkit.WebView
+import android.widget.LinearLayout
+import android.widget.Toast
+import kotlin.reflect.KFunction0
+
+
+class WebAppInterface(private val mContext: Context, var parentLayout: LinearLayout, var webView: WebView, var options: HashMap<String, KFunction0<Unit>>) {
+
+    // Show a toast from html.
+    @JavascriptInterface
+    fun showToast(toast: String) {
+        Toast.makeText(mContext, toast, Toast.LENGTH_SHORT).show()
+    }
+
+    // Exit the Immersive Reader.
+    @JavascriptInterface
+    fun immersiveReaderExit() {
+        webView.post(Runnable { destroyWebView(parentLayout, webView) })
+    }

```

Comment:
this is where to call add custom logic for what to do when the Immersive Reader exits